evaluation/metrics/metric_getters.py

```diff
@@ -25,12 +25,18 @@ def get_eval_metrics(task, predictions: list, targets: list) -> tuple:
     return score_dict[perf_metric_type], score_dict
 
 
-def get_job_metrics(job, dataset) -> dict:
+def get_job_metrics(job, dataset, decen=False) -> dict:
     if not job.aws_metrics:
         return {}
-    instance_config = instance_property[dataset.task.instance_type]
+
+    if decen:
+        instance_config = instance_property[dataset.task_instance_type]
```

Comment:
I believe this was because of how I converted the `Dataset` class into a dict: https://github.com/facebookresearch/dynabench/blob/decentralized_eaas/evaluation/datasets/common.py#L384

The reason was because I did not want to turn the entire task into a dict, when I only needed the instance type. I guess if we want to keep it the same, I could just make `task` map to a `dotdict` when I convert to a dict, so that the code here does not need to change?